tests/test_frame.py

```diff
@@ -1132,3 +1132,11 @@ def test_frame_reindex() -> None:
     # GH 84
     df = pd.DataFrame({"a": [1, 2, 3]}, index=[0, 1, 2])
     df.reindex([2, 1, 0])
+
+
+def test_frame_ndarray_assignmment() -> None:
```

Comment:
Can you add a reference to the Github issue?

Reply:
Added